htmd/charge/__init__.py

```diff
@@ -0,0 +1,6 @@
+# (c) 2015-2018 Acellera Ltd http://www.acellera.com
```

Comment:
Following the case of `ffevaluation` and `qm`, it should be outside. The long-term goal is to decompose `parameterize` into building blocks.

Reply:
fine, but charge module seems fairly general and not understandable. I believe. I don't have strong opinions though.

Reply:
@stefdoerr, suggestions?

Reply:
I don't see how it can be used outside the context of parameterization so it's useless in my opinion to HTMD as a general tool. Maybe if you demonstrate a usage of the charges module in HTMD outside the context of parameterization I could be convinced.

Reply:
Same for qm by the way which I consider also tied to parameterization.

Reply:
`ffevaluate` is used generally also by other projects as it has broader applications.

Reply:
Wait, let's move this discussion to slack better.